README.md

```diff
@@ -62,13 +62,25 @@ spec:
               mountPath: C:\var\lib\kubelet
             - name: plugin-dir
               mountPath: C:\csi
-            - name: csi-proxy-pipe
-              mountPath: \\.\pipe\csi-proxy-v1alpha1
+            - name: csi-proxy-disk-pipe
+              mountPath: \\.\pipe\csi-proxy-disk-v1alpha1
+            - name: csi-proxy-volume-pipe
+              mountPath: \\.\pipe\csi-proxy-volume-v1alpha1
+            - name: csi-proxy-filesystem-pipe
+              mountPath: \\.\pipe\csi-proxy-filesystem-v1alpha1
       volumes:
-        - name: csi-proxy-pipe
+        - name: csi-proxy-disk-pipe
           hostPath:
             path: \\.\pipe\csi-proxy-v1alpha1
```

Comment:
csi-proxy-disk-pipe ?

Reply:
I think the internal name could be anything but for clarity, adding the version may be great.